pyproject.toml

```diff
@@ -13,6 +13,7 @@ python = "^3.10"
 sib-api-v3-sdk = "^7.6.0"
 cryptography = "^41.0.1"
 python-dotenv = "^1.0.0"
+discord-py-pagination = "^0.0.5"
```

Comment:
not a fan of this, especially considering the version number, and the reputation of the maintainer.

Reply:
what do you suggest I do?

Reply:
DIY it, you can try converting [`discord-ext-menus`](https://github.com/Rapptz/discord-ext-menus) to use buttons.

Or I can do it for you, and send a PR to this branch.

Reply:
i made a commit using `discord-ext-menus`. would be awesome if you could test it

Reply:
check your PRs :)